src/Azure.Functions.Cli/Kubernetes/KubernetesHelpers.cs

```diff
@@ -140,7 +153,9 @@ internal static string GetKedaResources(string @namespace)
             int? cooldownPeriod = null,
             string serviceType = "LoadBalancer",
             int? minReplicas = null,
-            int? maxReplicas = null)
+            int? maxReplicas = null,
+            string keysSecretCollectionName = "func-keys-secret",
```

Comment:
The default name for the secret collection should drive from the deployment name to account for multiple deployments.

Reply:
also if you don't pass `--keys-secret-name` this will be null, which will cause the deployment to fail trying to create a resource with an empty name